census/README.md

```diff
@@ -77,14 +83,15 @@ There are two options for the virtual environments:
  * Install [gcloud](https://cloud.google.com/sdk/gcloud/)
  * Install the python dependencies. `pip install --upgrade -r requirements.txt`
 
-# Single Node Training
+# How to run the code
+You can run the code either as a stand-alone python program or using `gcloud`.
+
+Before you run any code, you should locally change your working directory to the appropriate subdirectory -- [./estimator](./estimator) for the prebuilt estimator, [./customestimator](./customestimator) for the custom Estimator-based model, or [./tensorflowcore](./tensorflowcore) for the core TensorFlow model.
```

Comment:
Nice change. Can you add these as bulleted items so its more clear. May be something like:

* In order to use Canned estimator `cd estimator`